deploy/nfs/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -39,6 +39,21 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["csinodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshots"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotcontents"]
+    verbs: ["create", "get", "list", "watch", "update", "delete", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotcontents/status"]
+    verbs: ["update", "patch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshots/status"]
+    verbs: ["update", "patch"]
```

Comment:
Please reverify once do we need all these access?

Reply:
> Please reverify once do we need all these access?

okay, I am copying these permissions
https://github.com/kubernetes-csi/external-snapshotter/blob/v5.0.1/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml#L33-L41